examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -85,13 +85,14 @@ CommissioningParameters PairingCommand::GetCommissioningParameters()
 
 CHIP_ERROR PairingCommand::PaseWithCode(NodeId remoteId)
 {
-    return CurrentCommissioner().EstablishPASEConnection(remoteId, mOnboardingPayload);
+    return CurrentCommissioner().EstablishPASEConnection(remoteId, mOnboardingPayload, mUseOnlyOnNetworkDiscovery.ValueOr(false));
 }
 
 CHIP_ERROR PairingCommand::PairWithCode(NodeId remoteId)
 {
     CommissioningParameters commissioningParams = GetCommissioningParameters();
-    return CurrentCommissioner().PairDevice(remoteId, mOnboardingPayload, commissioningParams);
+    return CurrentCommissioner().PairDevice(remoteId, mOnboardingPayload, commissioningParams,
+                                            mUseOnlyOnNetworkDiscovery.ValueOr(false));
```

Comment:
Shouldn't this default to "we have no network credentials" instead of false?  That is, by default we'd wat to do only on-network if we have no network credentials in the commissioningParams.

Also, don't we need to pass `true` for this boolean in the pairing calls YAML makes?

Reply:
> Shouldn't this default to "we have no network credentials" instead of false? That is, by default we'd wat to do only on-network if we have no network credentials in the commissioningParams.
> 

Ah that a good idea. We will keep the behaviour from the previous patch then.

> Also, don't we need to pass `true` for this boolean in the pairing calls YAML makes?

We do... I will fix that.